cookbooks/aws-parallelcluster-platform/test/controls/ssh_spec.rb

```diff
@@ -21,6 +22,18 @@
   end
 end
 
+control 'ssh_target_checker_script_created_ec2' do
```

Comment:
Could you remove duplicated code and just add the condition "if on docker" to the line which works only on docker? Something like:

```
on_docker = os_properties.on_docker?
...
its('content') { should match /VPC_CIDR_LIST=\(cidr1 cidr2\)/ } if on_docker
```